provisioning/roles/apache/tasks/Linux.yaml

```diff
@@ -0,0 +1,14 @@
+- name: Install apache server
```

Comment:
Rename all these files to lowercase name

Reply:
Done 7b23130e54538bb5b067ffead8342c73ad136320